docs/react-native-devtools.md

```diff
@@ -9,7 +9,7 @@ React Native DevTools is our new debugging experience featuring an end-to-end re
 
 ![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
 
-React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](/docs/debugging-native-code)).
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](https://reactnative.dev/docs/next/debugging-native-code).
```

Comment:
Hello @HammzaHM, nice catch! 👍 

Looks like duplicated bracket character at the end of URL results in broken link generation. It should be fine to stick to the relative URL and current docs version:
```suggestion
React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](/docs/debugging-native-code).
```

Reply:
Hello @Simek, thanks for your review.

The relative URL in the current docs version redirected me to this link 👉🏻`http://localhost:3000/docs/next/debugging-native-code` as shown in the screenshot below, and there is no duplicated bracket, there is an opened one after the word `Xcode` “(see ... (/docs/debugging-native-code)).“,


![Screenshot 2024-11-06 at 10 02 39 AM](https://github.com/user-attachments/assets/699ecb08-f86a-4700-a0c3-7511ceefa25e)

Reply:
Yup, I think that the closing bracket confuses Docusaurus leading to this weird Href. Removing additional bracket should fix the problem, since we are using relative links across all of the content pages.

Reply:
@Simek Okay, Thank you.